zk-kyc/zkvm/methods/README.md

```diff
@@ -0,0 +1,5 @@
+# Methods
+
+This directory defines the guest source code to be executed by the zkVM on Bonsai. The guest program uses the [`zkvm/oidc-validator`] library to validate the integrity of an OIDC issued JWT against a providers RS256 public key.  
+
+[`zkvm/oidc-validator`]: ../zkvm/oidc-validator
```

Comment:
hmm -- i'm ending up at this path when i click this link in the preview. 

`zk-kyc/zkvm/zkvm/oidc-validator`